### ISSUE TITLE
FreeBSD - ZFS mounted drive

### ISSUE THREAD

**Issue by steverob** [rjeczalik/notify#111]
Hi, I am trying to use `notify` to listen to `CREATE` events in a volume mounted over ZFS in a server running FreeBSD.

But the event does not get captured at all. The code works fine in my Ubuntu distribution.

Any idea why? Thanks!

**Comment by pblaszczyk** [comment]
Hi. Please show your code + output of `mount` + used commands with information on which mount point they are executed + output of `freebsd-version`.

**Comment by steverob** [comment]
@pblaszczyk sure. Here is the code - 

```go
package main

import (
  "github.com/rjeczalik/notify"
  "fmt"
  "os"
)

func init() {
  raven.SetDSN("XYZ")
}

func main() {

  c := make(chan notify.EventInfo, 10)
  watchRoot := os.Args[1]

  if err := notify.Watch(watchRoot + "/...", c, notify.Create); err != nil {
    raven.CaptureError(err, nil)
  }
  defer notify.Stop(c)

  for {
    fmt.Println("Waiting...")
    ei := <-c
    fmt.Println("Received: ", ei)
  }
}
```

**Comment by steverob** [comment]
@pblaszczyk any thoughts on this? Appreciate your effort!

**Comment by pblaszczyk** [comment]
Hi. Yes. Unfortunately I believe there is a bug here. Just to confirm that this is really the same scenario: please try watching on events `notify.Create | notify.Write` and check if with this configuration `notify.Create` events are triggered.

**Comment by rjeczalik** [comment]
Just fyi - debug logs may also be helpful, build/run your code with debug tag:

```
$ go run -tags debug main.go
```

**Comment by steverob** [comment]
@pblaszczyk Thanks, I'll try with that and let you know.

@rjeczalik unfortunately in the target machine I cannot install go. :(

**Comment by steverob** [comment]
@pblaszczyk thanks. THAT WORKS! :) Thanks a lot.

**Comment by pblaszczyk** [comment]
Closed by #112.
